src/platform/pdf/registerFonts.js

```diff
@@ -26,7 +27,7 @@ const downloadAndRegisterFont = async (doc, font) => {
   const bucket = environment.isLocalhost()
     ? ''
     : BUCKETS[environment.BUILDTYPE];
-  const request = await fetch(`${bucket}/generated/${knownFonts[font]}`);
+  const request = await fetch(`${bucket}/fonts/${knownFonts[font]}`);
```

Comment:
This is enough of a change that I'd like to see this PR go in late afternoon or early morning so that we can test on staging before it goes out to prod.

Reply:
Definitely, I'll be sure to merge one of those times.